### PR TITLE
Remove generated API docs from the README

### PR DESCRIPTION
**Pull request by imaustink**
This pull request was created with [Landscaper](https://github.com/bitovi/landscaper).
The following code mods were used to create this PR:

1. **https://gist.github.com/imaustink/f22e65791d3b874680b6c77550bebd32**,1. **https://gist.github.com/imaustink/dc8c702f9940cc3346cc41338e890838**,1. **https://gist.github.com/imaustink/72693934627c9fcc24ffbbb6b5163929**

Please review this PR carefully as Landscaper does not guarantee any code mod's quality.

### CODE REVIEW

**Review by chasenlehara — CHANGES_REQUESTED** [review]
We should finish https://github.com/canjs/can-observe/issues/7 before merging this.

**Comment by imaustink** [comment]
At @chasenlehara, I opened a PR (https://github.com/canjs/canjs/pull/3504) to close #7. Should I wait until it's merged to merge this?

**Pull request merged**
